cmd/crates/soroban-test/tests/it/integration/tx/operations.rs

```diff
@@ -0,0 +1,414 @@
+    assert_eq!(seq, after.seq_num);
+}
+
+#[tokio::test]
+async fn account_merge() {
+    let sandbox = &TestEnv::new();
+    let client = soroban_rpc::Client::new(&sandbox.rpc_url).unwrap();
+    let (test, test1) = setup_accounts(sandbox);
+    let before = client.get_account(&test).await.unwrap();
+    let before1 = client.get_account(&test1).await.unwrap();
+    let fee = 100;
+    sandbox
+        .new_assert_cmd("tx")
+        .args([
+            "new",
+            "account-merge",
+            "--source",
+            "test1",
+            "--account",
+            test.as_str(),
+            "--fee",
+            fee.to_string().as_str(),
+        ])
+        .assert()
+        .success();
+    let after = client.get_account(&test).await.unwrap();
+    assert!(client.get_account(&test1).await.is_err());
+    assert_eq!(before.balance + before1.balance - fee, after.balance);
+}
+
+#[tokio::test]
+async fn set_trustline_flags() {
+    let sandbox = &TestEnv::new();
+    let client = soroban_rpc::Client::new(&sandbox.rpc_url).unwrap();
+    let (test, issuer) = setup_accounts(sandbox);
+    let before = client.get_account(&test).await.unwrap();
+    let asset = format!("usdc:{issuer}");
+    issue_asset(sandbox, &test, &issuer, &asset, 100_000, 100).await;
+    let after_issue = client.get_account(&test).await.unwrap();
+    // set trustline flags tx new
+    let res = sandbox
+        .new_assert_cmd("tx")
+        .args([
+            "new",
+            "set-trustline-flags",
+            "--asset",
+            &asset,
+            "--trustor",
+            &test,
+            "--set-authorize",
+            "--source",
+            "test1",
+        ])
+        .assert()
+        .success()
+        .stdout_as_str();
+    let after = client.get_account(&test).await.unwrap();
+    println!("{before:#?}\n{after_issue:#?}\n{after:#?}");
+    println!("{res}");
+}
+
+#[tokio::test]
+async fn set_options_add_signer() {
+    let sandbox = &TestEnv::new();
+    let client = soroban_rpc::Client::new(&sandbox.rpc_url).unwrap();
+    let (test, test1) = setup_accounts(sandbox);
+    let before = client.get_account(&test).await.unwrap();
+    sandbox
+        .new_assert_cmd("tx")
+        .args([
+            "new",
+            "set-options",
+            "--signer",
+            test1.as_str(),
+            "--signer-weight",
+            "1",
+        ])
+        .assert()
+        .success();
+    let after = client.get_account(&test).await.unwrap();
+    assert_eq!(before.signers.len() + 1, after.signers.len());
+    // Now remove signer with a weight of 0
+    sandbox
+        .new_assert_cmd("tx")
+        .args([
+            "new",
+            "set-options",
+            "--signer",
+            test1.as_str(),
+            "--signer-weight",
+            "0",
+        ])
+        .assert()
+        .success();
+    let after = client.get_account(&test).await.unwrap();
+    assert_eq!(before.signers.len(), after.signers.len());
```

Comment:
Requesting the appropriate ledger entry it should be possible to retrieve the added signer too and use that in an assertion.